chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/AuthenticationController.java

```diff
@@ -271,23 +269,36 @@ public String resetPassword(@RequestBody final UserResetPasswordRequest userInfo
         return "{\"passwordResetEmailSent\" : true }";
     }
 
+    @Deprecated
     @ApiOperation(value = "Impersonate another user.", notes = "")
```

Comment:
Can you add "DEPRECATED" to the text here? Done in a few other controllers with deprecated methods